### ISSUE TITLE
Changing infiniteScrollDisabled doesn't seem to work

### ISSUE THREAD

**Issue by dylan-smith** [orizens/ngx-infinite-scroll#195]
If I bind [infiniteScrollDisabled] to a property in my component, and it defaults to true, then I later change it (e.g. when my list is done populating with initial data), the infiniteScroll event never gets triggered, as if the directive never actually re-enabled itself when the input property changed.

I fixed it in the js from the npm package, I assume it's a similar fix in the source TS code.  I changed:

```
InfiniteScrollDirective.prototype.ngOnChanges = function (_a) {
    var infiniteScrollContainer = _a.infiniteScrollContainer;
    if (inputPropChanged(infiniteScrollContainer)) {
        this.destroyScroller();
        this.setup();
    }
};
```
to
```
InfiniteScrollDirective.prototype.ngOnChanges = function (_a) {
    var infiniteScrollContainer = _a.infiniteScrollContainer;
    var infiniteScrollDisabled = _a.infiniteScrollDisabled;
    if (inputPropChanged(infiniteScrollContainer) || inputPropChanged(infiniteScrollDisabled)) {
        this.destroyScroller();
        this.setup();
    }
};
```

**Comment by orizens** [comment]
did you try the latest 0.6.1 version? 
Also - i was thinking of creating/destroying the scroll based on this prop - but need to think of the usecase of disabling. will it be better to recreate or just really disable?

**Comment by dylan-smith** [comment]
I was using the latest - just checked and it's 0.6.1.

**Comment by orizens** [comment]
should be fixed after #196 is released

**Comment by orizens** [comment]
added with #196 and available with v 0.7.0
